src/mockServer/msw/server.ts

```diff
@@ -17,7 +17,7 @@ export const mockServiceWorkerServer = ({
   dbName?: string;
 }): { worker: SetupWorker; resetDB: (data: Database) => void } => {
   const fullAppContext = buildMockLocalContext(appContext);
-  const mswMocks = buildMSWMocks(fullAppContext, database, dbName);
+  const mswMocks = buildMSWMocks(fullAppContext, database, fullAppContext, dbName);
```

Comment:
I think using the same variable twice add confusion. My comment about the buildMSWMocks could maybe solve this.